### ISSUE TITLE
\cref and \Cref not recognized

### ISSUE THREAD

**Issue by samuelstjean** [sylvainhalle/textidote#17]
Hello,

Seems like the commands from cleveref [1] are not properly detected, I get errors about unreferenced figures such as 
~~~
* L364C12-L618C34 Figure fig:resampling_example is never referenced in the 
  text [sh:figref] 
      \label{fig:resampling_example}
             ^^^^^^^^^^^^^^^^^^^^^^^
~~~

but they are properly referenced by cleveref, producing a false positive. It's probably missing a quick filter for matching the cleveref equivalent of \ref.

[1] https://ctan.org/pkg/cleveref

**Comment by sylvainhalle** [comment]
Exactly. We simply need to match `\cref` and `\Cref` in addition to `\ref`. Will slate this for the next release!
